test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2691,12 +2691,17 @@ def forward(self, input, other):
         y = torch.randn(6, 4)
         self.run_test(ViewModel(), (x, y))
 
-    @disableScriptTest()  # ONNX Shape inference failure in if/else block for Gemm
```

Comment:
can you also enable tests: test_split_tensor_scalar, test_interpolate_function_substitution and test_groupnorm_noaffine for scripting?